src/std/math.ab

```diff
@@ -0,0 +1,5 @@
+#[allow_absurd_cast]
+pub fun sum(list: [Num]): Num {
```

Comment:
I think that this function should be in `std/array`

Reply:
We have a PR ready for Math functions so I was preparing for that.
Also sum seems a math method, also if it involve an array.

Reply:
Hmm okay